inc/saiacl.h

```diff
@@ -354,6 +354,15 @@ typedef enum _sai_acl_table_group_attr_t
      */
     SAI_ACL_TABLE_GROUP_ATTR_MEMBER_LIST,
 
+    /**
+     * @brief Match type for the table group
+     *
+     * @type sai_acl_match_type_t
+     * @flags CREATE_ONLY
+     * @default SAI_ACL_MATCH_TYPE_PRIORITY_BASED
+     */
+    SAI_ACL_TABLE_GROUP_ATTR_ACL_MATCH_TYPE,
```

Comment:
As discussed during the meeting, we should 

*   remove this attribute and 
*   add the switch acl table capability to specify whether the platform can support mixing Exact and Ternary match in the same ACL Table Group.

Reply:
Suggested changes are incorporated in the latest diffs